src/server/controllers/movie.js

```diff
@@ -1,31 +1,48 @@
-const jwt = require('jsonwebtoken');
-const { PrismaClient } = require('@prisma/client');
+const jwt = require("jsonwebtoken");
+const { PrismaClient } = require("@prisma/client");
 const prisma = new PrismaClient();
 
-const jwtSecret = 'mysecret';
+const jwtSecret = "mysecret";
 
 const getAllMovies = async (req, res) => {
-    const movies = await prisma.movie.findMany();
-
-    res.json({ data: movies });
+  const movies = await prisma.movie.findMany();
+  res.json({ data: movies });
+  console.log('zzzzzzzzzzzzzzzz', movies);
```

Comment:
Love the use of a comment in a console log so you can easily spot what you're looking for!